website/source/docs/connect/proxies/envoy.md

```diff
@@ -216,10 +216,27 @@ definition](/docs/connect/registration/service-registration.html) or
 
 - `protocol` - Same as above in main config but affects the listener setup for
   the upstream.
+
+    -> **Note:** The protocol of a service should ideally be configured via the
+    [`protocol`](/docs/agent/config-entries/service-defaults.html#protocol)
+    field of a
+    [`service-defaults`](/docs/agent/config-entries/service-defaults.html)
+    config entry for the upstream destination service. Configuring it in a
```

Comment:
Is there an example of when someone would want to configure protocol here instead?

Reply:
This (and the other example you called out below) is a backwards compatibility issue mostly.

Reply:
Sweet. Let's call that out so we don't leave people wondering?

Reply:
Fixed.